frontend/src/components/contextMenu/model/ContextMenuModel.ts

```
@@ -0,0 +1,84 @@
+  }
+
+  get anchorVar(): string {
+    return `--${this.htmlId}`;
+  }
+
+  get openedGroup(): ContextGroupModel | undefined {
+    return this.state.openedGroup;
+  }
+
+  executeAction(action: ContextActionModel): void {
+    action.onAction({ menuRect: this.menuRect! });
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)